### PR TITLE
Serialization Context Changes

### PR DESCRIPTION
**Pull request by ryanemerson**
https://issues.redhat.com/browse/ISPN-13041
https://issues.redhat.com/browse/ISPN-13042

### CODE REVIEW

**Comment by ryanemerson** [comment]
`core/proto.lock` update to reflect this change. Assuming that the UUIDAdaptor TypeID is fixed in protostream, this won't lead to any backward incompatibilities. This PR shouldn't be merged until the protostream update is available.

**Comment by ryanemerson** [comment]
The `ProtoStreamBackwardsCompatibilityTest` failures should be fixed once the UUIDAdaptor TypeID is fixed in protostream.

**Review by wburns — APPROVED** [review]
LGTM, just waiting on CI.

**Comment by wburns** [comment]
This test failure looks related
https://ci.infinispan.org/job/Infinispan/job/PR-9311/5/testReport/org.infinispan.marshall.protostream.impl/ProtoStreamBackwardsCompatibilityTest/testOldEventLoggerUUIDBytesAreReadable/

**Comment by wburns** [comment]
Integrated into master, thanks @ryanemerson !

File: core/src/main/java/org/infinispan/marshall/protostream/impl/SerializationContextRegistryImpl.java [hunk 1]
```diff
@@ -57,18 +57,14 @@ public void start() {
 
       String messageName = PersistenceContextInitializer.getFqTypeName(MarshallableUserObject.class);
       BaseMarshaller userObjectMarshaller = new MarshallableUserObject.Marshaller(messageName, userMarshaller.wired());
-      update(GLOBAL, ctx -> ctx.addContextInitializer(commonTypesSchema)
-            .addContextInitializer(commonContainerTypesSchema)
-            .addContextInitializer(new PersistenceContextInitializerImpl())
+      update(GLOBAL, ctx -> ctx.addContextInitializer(new PersistenceContextInitializerImpl())
```
Comment (danberindei) [inline core/src/main/java/org/infinispan/marshall/protostream/impl/SerializationContextRegistryImpl.java:60]: Shouldn't this have been `new GlobalContextInitializerImpl`?

Comment (ryanemerson) [reply core/src/main/java/org/infinispan/marshall/protostream/impl/SerializationContextRegistryImpl.java:60]: No, a `GlobalContextInitializerImpl` does not exist in core. Currently there is only `org.infinispan.commons.GlobalContextInitializerImpl()`

**Pull request merged**